source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst

```diff
@@ -103,6 +103,15 @@ Local JBOD Storage with Sequential Mounts
    For deployments that *require* using network-attached storage, use
    NFSv4 for best results.
 
+Time Synchronization
+~~~~~~~~~~~~~~~~~~~~
+
+Multi-node systems must maintain synchronized time and date, as reads and writes happen across nodes.
```

Comment:
I would make this a little more generic

```suggestion
Multi-node systems must maintain synchronized time and date to maintain stable internode operations and interactions."
```

IIRC there are non-read/write ops that could be affected by clock skew, such as TLS connections.

Reply:
Done.